### PR TITLE
Updated Frame Component

### LINKED ISSUE #80: Create Frame Component
image

- Create a image frame component that will take in a square image and put it in the gray area

### PR DESCRIPTION
**Pull request by tonytrieu-dev**

### CODE REVIEW

File: components/Frame.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
 import React from "react";
 
-const Frame = () => {
-	return <div>Frame</div>;
+const Frame = ({ squareImage }) => {
+	return (
+		<img
+			className='ml-5 border-solid'
+			src={squareImage}
+			alt='Square image'
```
Comment (shahdivyank) [inline components/Frame.js]: Rename to `ACM Gallery Picture`

File: components/Frame.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import React from "react";
 
-const Frame = () => {
-	return <div>Frame</div>;
+const Frame = ({ squareImage }) => {
```
Comment (shahdivyank) [inline components/Frame.js]: call the prop, `image`

File: tailwind.config.js [hunk 3]
```diff
@@ -13,6 +13,9 @@ module.exports = {
 			backgroundImage: {
 				arrow: "url('/chevron.png')",
 			},
+			borderImage:
+				"conic-gradient(from 10deg, marine 70deg, black 0deg, black 175deg, lightpurple 175deg, lightpurple 255deg, black 3deg)",
```
Comment (shahdivyank) [inline tailwind.config.js]: use the color signatures from the `colors` object below instead of the defaults

File: components/Frame.js [hunk 4]
```diff
@@ -1,7 +1,13 @@
...
+	return (
+		<img
+			className='ml-5 border-solid'
```
Comment (shahdivyank) [inline components/Frame.js]: what is the `ml-5` for?

**Review by shahdivyank — CHANGES_REQUESTED** [review]

**Comment by shahdivyank** [comment]
Can you also assign yourself the PR

File: components/Frame.js (hunk 4 above)
Comment (tonytrieu-dev) [reply components/Frame.js]: Oh, you added it before. I forgot to remove it. My bad.

File: tailwind.config.js (hunk 3 above)
Comment (tonytrieu-dev) [reply tailwind.config.js]: Okay.

File: components/Frame.js (hunk 2 above)
Comment (tonytrieu-dev) [reply components/Frame.js]: Alright alright alright

File: components/Frame.js (hunk 1 above)
Comment (tonytrieu-dev) [reply components/Frame.js]: The component?

**Review by shahdivyank — APPROVED** [review]

**Review by vwong031 — APPROVED** [review]

**Pull request merged**